sample/src/main/java/com/polidea/rxandroidble/sample/example4_characteristic/advanced/AdvancedCharacteristicOperationExampleActivity.java

```diff
@@ -1,241 +0,0 @@
-package com.polidea.rxandroidble.sample.example4_characteristic.advanced;
-
-import android.os.Bundle;
-import android.support.annotation.NonNull;
-import android.support.annotation.StringRes;
-import android.support.design.widget.Snackbar;
-import android.util.Log;
-import android.view.View;
-import android.widget.Button;
-import android.widget.TextView;
-import butterknife.BindView;
-import butterknife.ButterKnife;
-import com.jakewharton.rxbinding.view.RxView;
-import com.polidea.rxandroidble.RxBleDevice;
-import com.polidea.rxandroidble.sample.DeviceActivity;
-import com.polidea.rxandroidble.sample.R;
-import com.polidea.rxandroidble.sample.SampleApplication;
-import com.polidea.rxandroidble.sample.util.HexString;
-import com.trello.rxlifecycle.components.support.RxAppCompatActivity;
-import java.util.UUID;
-import rx.Observable;
-import rx.Subscription;
-import rx.android.schedulers.AndroidSchedulers;
-
-/**
- * This activity allows for connecting to a device and interact with a given characteristic.
- * <p>
- * It may be used as a direct replacement for
- * {@link com.polidea.rxandroidble.sample.example4_characteristic.CharacteristicOperationExampleActivity}
- * <p>
- * When the connection is not established only the "CONNECT" button is active.
- * When the user will click on "CONNECT" button the connection is established and other buttons are enabled according to the properties
- * of the characteristic.
- * Subsequent clicks on "CONNECT" button (which name will change accordingly to the connection state) will close the connection.
- * <p>
- * When the connection is open other buttons are activated in regards of the characteristic's properties.
- * If the user clicks on "READ" a characteristic read is performed and the output is set on the TextView and EditText or a Snackbar is shown
- * in case of an error. "WRITE" clicks work the same as read but a write command is performed with data from the EditText.
- * If the characteristic has both of PROPERTY_NOTIFY and PROPERTY_INDICATE then only one of them is possible to be set at any given time.
- * Texts on notification and indication buttons will change accordingly to the current state of the notifications / indications.
- */
-public class AdvancedCharacteristicOperationExampleActivity extends RxAppCompatActivity {
```

Comment:
Why removed? It was meant as a showcase for those who would like to see some more advanced usage. There are references to this file on Stackoverflow for instance.

Reply:
Correct, my mistake. Removed due to some temporary compilation issues.